PoCs/README.md

```diff
@@ -0,0 +1,139 @@
+# PoCs
+
+This folder is dedicated to hosting various PoCs for testing and demonstrating
+potential configurations or enhancements in the Metal3 environment.
+
+## PoCs Overview
+
+### 1. PoC Hostnetworkless Ironic with VirtualMedia Using NodePorts
+
+This PoC is part of the discussion [metal3-io/baremetal-operator#
+1739](https://github.com/metal3-io/baremetal-operator/discussions/1739) and
+demonstrates running Ironic without `hostNetwork`, limited to the VirtualMedia
+use case, using a NodePort service.
+
+**Branch:**
+
+Run this PoC or review code changes at [hostnetworkless-ironic-
+nodeport](https://github.com/Nordix/metal3-dev-env/tree/test-svc-nodePort/mboukhalfa)
+
+**Key Changes:**
+
+1. **Edit Ironic Deployment:**
+
+   - Remove `hostNetwork: true`.
+   - Remove `dnsmasq` container.
+   - Remove security restrictions to allow root access for debugging inside
+   the containers.
+
+2. **Add NodePort Service:**
+
+   - Map the following ports:
+
+   ```pseudocode
+   - ironic
+     30085:6385
+   - inspector
+     30050:5050
+   - httpd
+     30080:30080
+   ```
+
+   - Changed the `httpd` internal port because the `HTTP_PORT` variable is used
+   in both internal and external configurations.
+
+3. **Edit Ironic ConfigMap:**
+
+   - Remove `PROVISIONING_IP` so that `runironic` uses the pod's IP from the
+   `eth0` interface.
+   - Add external Ironic IPs to be published to external components like IPA.
+
+4. **Provisioning Networks:**
+
+   - Remove `keepalived` since it cannot access the `ironicendpoint` bridge; instead,
+   use manual commands to configure the bridge with the Ironic external IP.
+   - Manually remove the IP from minikube and add it to the CP node during pivot.
+
+### 1. PoC Hostnetworkless Ironic with VirtualMedia Using MetalLB LoadBalancer
+
+This PoC is part of the discussion [metal3-io/baremetal-operator#1739](https
+://github.com/metal3-io/baremetal-operator/discussions/1739) and
+demonstrates running Ironic without`hostNetwork`, limited to the
+VirtualMedia use case, using a MetalLB LoadBalancer service.
+
+**Branch:**
+
+Run this PoC or review code changes at [hostnetworkless-ironic-metallb](https
+://github.com/Nordix/metal3-dev-env/tree/PoC-lb-ironic/mohammed)
+
+**Key Changes:**
+
+1. **Edit Ironic Deployment:**
+
+   - Remove `hostNetwork: true`.
+   - Remove `dnsmasq` container.
+   - Remove `keepalived` container.
+   - Remove security restrictions to allow root access for debugging inside
+   the containers.
+
+2. **Add MetalLB Service:**
```

Comment:
There auto-numbering lists do work, when indentations in between are done right (like they are now).

```suggestion
1. **Add MetalLB Service:**
```